x-pack/docs/en/security/troubleshooting.asciidoc

```diff
@@ -710,6 +710,25 @@ If you want your users to be able to use local credentials to authenticate to
 {kib} in addition to using the SAML realm for Single Sign-On, you must enable
 the `basic` `authProvider` in {kib}. The process is documented in the
 <<saml-kibana-basic, SAML Guide>>
+
+--
+
+. *Symptoms:*
++
+--
+No response ID values are being passed from {kib} to {es}
```

Comment:
__nit:__ the issue here is that Kibana sends only SAML response to ES without accompanying SAML request ID since we lost session cookie (SAML request ID is stored in the session).  
```suggestion
No SAML request ID values are being passed from {kib} to {es}:
```

Reply:
++